### PR TITLE
chore: updating windows vhds for v0.70.0

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/azenvtypes.go [hunk 1]
```diff
@@ -138,18 +138,18 @@ var (
...
 
 	// AKSWindowsServer2019ContainerDOSImageConfig is the aks-engine image based on Windows Server 2019
 	// configured with containerd
 	AKSWindowsServer2019ContainerDOSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-ctrd-2203",
+		ImageSku:       "2019-datacenter-core-ctrd-2204",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.2565.220301",
+		ImageVersion:   "17763.2565.220407",
```
Comment (jackfrancis) [inline pkg/api/azenvtypes.go:152]: I assume it's intentional that these two images have a different image version? (built on subsequent days?)

Comment (marosset) [reply pkg/api/azenvtypes.go:152]: Kind of... the CI job which added the images to the marketplace office ran on different days for the two images and that timestamp is generated by the script that does the publishing.

Comment (marosset) [reply pkg/api/azenvtypes.go:152]: so not intentional but by design?

Comment (jackfrancis) [reply pkg/api/azenvtypes.go:152]: All good, thanks for clarifying. Programmatic naming is good. :)

**Pull request merged**